pkg/operator2/route.go

```diff
@@ -21,7 +21,6 @@ func (c *osinOperator) handleRoute() (*routev1.Route, error) {
 	if len(route.Spec.Host) == 0 {
 		return nil, fmt.Errorf("route has no host: %#v", route)
 	}
-	// TODO make sure route is correct
```

Comment:
This means validate the route from the API, which we still do not do.

Reply:
Oh boy, I thought this was a weird request the way I grasped it
![figure](https://memegenerator.net/img/images/300x300/5710564.jpg)

Reply:
The message you had in your git commit would be reasonable comment.

Reply:
Found it in reflog, I'll add it back with next push.